src/Core/Models/Api/Response/OrganizationSponsorships/OrganizationSponsorshipResponseModel.cs

```diff
@@ -1,19 +1,28 @@
 ﻿using System;
+using System.Text.Json.Serialization;
 using Bit.Core.Enums;
 using Bit.Core.Models.Data.Organizations.OrganizationSponsorships;
 
 namespace Bit.Core.Models.Api.Response.OrganizationSponsorships
 {
     public class OrganizationSponsorshipResponseModel
     {
+        [JsonPropertyName("SponsoringOrganizationUserId")]
         public Guid SponsoringOrganizationUserId { get; set; }
+        [JsonPropertyName("FriendlyName")]
         public string FriendlyName { get; set; }
+        [JsonPropertyName("OfferedToEmail")]
         public string OfferedToEmail { get; set; }
+        [JsonPropertyName("PlanSponsorshipType")]
         public PlanSponsorshipType PlanSponsorshipType { get; set; }
+        [JsonPropertyName("LastSyncDate")]
         public DateTime? LastSyncDate { get; set; }
+        [JsonPropertyName("ValidUntil")]
         public DateTime? ValidUntil { get; set; }
+        [JsonPropertyName("ToDelete")]
         public bool ToDelete { get; set; }
 
+        [JsonPropertyName("CloudSponsorshipRemoved")]
         public bool CloudSponsorshipRemoved { get; set; }
```

Comment:
I don't understand why you need these, they're all just the same name

Reply:
The sync couldn't deserialize this response model and kept returning `null` for the list of sponsorships. I couldn't figure out a way to get it without the property names. Maybe I'm missing something?

Reply:
I'm more familiar with Newtonsoft than `system.Text.Json`, but my understanding is that these shouldn't be necessary, @justindbaur do you have any insight as to why these models are deserializing?